packages/block-library/src/image/image.js

```diff
@@ -694,7 +698,7 @@ export default function Image( {
 					setAttributes( {
 						width: `${ elt.offsetWidth }px`,
 						height: 'auto',
-						aspectRatio: `${ ratio }`,
+						aspectRatio: `${ stringAspectRatio }`,
```

Comment:
The `` `${ }` `` should have cast it as a string and `'1.618'`, for example, should be a valid string.

Reply:
I updated it to use `String()` instead so it's a little more visible.